include/sph/equationOfState.hpp

```diff
@@ -27,7 +27,7 @@ void computeEquationOfState(size_t startIndex, size_t endIndex, Dataset& d)
     T* cv   = d.cv.data();
 
 #pragma omp parallel for schedule(static)
-    for (size_t i = 0; i < endIndex; ++i)
+    for (size_t i = startIndex; i < endIndex; ++i)
```

Comment:
Good catch.

Reply:
(Note, your suggestion is incorrect, the loop has to go from `0` to `numParticles`, not just `endIndex`)
I'll also address this in a separate PR.